swagger/api/v1/presentation.yaml

```diff
@@ -0,0 +1,17 @@
+get:
+  operationId: presentation
```

Comment:
The operationId is not in line with the naming of other operaionIds that contain get.../create.... etc